integration-tests/plugins/simplest/tljh_simplest.py

```diff
@@ -39,4 +42,22 @@ def tljh_config_post_install(config):
 
 @hookimpl
 def tljh_custom_jupyterhub_config(c):
-    c.JupyterHub.authenticator_class = 'tmpauthenticator.TmpAuthenticator'
\ No newline at end of file
+    c.JupyterHub.authenticator_class = 'tmpauthenticator.TmpAuthenticator'
+
+
+@hookimpl
+def tljh_post_install():
```

Comment:
Can this test be something much simpler, like writing a random token to a file and verifying that? :D Just wanted to keep test runtime and complexity down

Reply:
Absolutely, should be fixed by https://github.com/jupyterhub/the-littlest-jupyterhub/pull/389/commits/b96a760aee1c39038ea743d5f3c540dd6ceed5b4.